docs/contrib/PlatformBringUp.md

```diff
@@ -162,35 +163,19 @@ export PLACE_DENSITY     = 0.70
 ```
 
 
-Refer to the [OpenSTA][https://github.com/The-OpenROAD-Project/OpenSTA/blob/master/doc/OpenSTA.pdf] documenation to change these variables to suit the needs of the specific design being made.
-
-
 ## constraint.sdc
 
-The `constraint.sdc` file defines timing constraints for the design. Here’s an example of a constraint.sdc file which defines a clock `clk` with a period of 8.4 nanoseconds.
+The `constraint.sdc` file defines timing constraints for the design. The `create_clock` command allows you to define clocks that are either connected to nets or are virtual and can be customized. The units for `create_clock` are in nanoseconds. Here’s an example of a `constraint.sdc` file which defines a clock `clk` with a period of 8.4 nanoseconds.
```

Comment:
The units for create_clock are not nanoseconds, but the liberty time units (advanced nodes will usually be in ps).

Reply:
Clarified that the units need to be consistent with the Liberty time units.